packages/core/src/TolgeeConfig.ts

```diff
@@ -17,10 +17,31 @@ export class TolgeeConfig {
   apiKey?: string;
   inputPrefix?: string = '%-%tolgee:';
   inputSuffix?: string = '%-%';
+  /**
+   * Overrides all language settings
+   */
   forceLanguage?: string;
+  /**
+   * Overrides language auto detection
+   */
   defaultLanguage?: string = 'en';
```

Comment:
It doesn't override. It is just default when if autodetection cannot be applied.